espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ScreenshotsHelper.java

```diff
@@ -46,16 +47,10 @@ public String getScreenshot() throws AppiumException {
             throw new ElementNotVisibleException("Cannot get a screenshot of the invisible view");
         }
 
-        view.setDrawingCacheEnabled(true);
-        final Bitmap bitmapScreenCap;
-        try {
-            bitmapScreenCap = Bitmap.createBitmap(view.getDrawingCache());
-            if (bitmapScreenCap == null) {
-                throw new ScreenCaptureException("Screen capture is impossible on the current view");
-            }
-        } finally {
-            view.setDrawingCacheEnabled(false);
-        }
+
+        final ScreenCapture screenCap = Screenshot.capture();
+        final Bitmap bitmapScreenCap = screenCap.getBitmap();
```

Comment:
this won't work as expected. I'd rather set `this.view` to null if we want to make full-screen shot and keep the old approach in case we want to take screenshot of a view